k9mail/src/main/java/com/fsck/k9/ui/messageview/MessageViewFragment.java

```diff
@@ -685,15 +651,42 @@ public void refreshAttachmentThumbnail(AttachmentViewInfo attachment) {
         // mMessageView.refreshAttachmentThumbnail(attachment);
     }
 
-    @Override
-    public void onPgpSignatureButtonClick(PendingIntent pendingIntent) {
-        try {
+    MessageCryptoMvpView messageCryptoMvpView = new MessageCryptoMvpView() {
```

Comment:
move where? it's essentially just methods of the enclosing class in a named scope, this seems like the place for those to be. but yes, should be private.

(I like this type of named scoping a lot more than implementing 5+ interfaces in classes which are already large and it becomes difficult to see what belongs where, plus methods tend to shift around now and then and become very mixed up after some time)